From: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
Date: Wed, 24 Feb 2016 20:02:25 +0100
Subject: repobuild: Allow customizing fuel-mirror repo URL

For Armband, fuel-mirror code needs to be patched outside of the build
system, so we factor out the repo URL into an overrideable variable
(FUEL_MIRROR_URL).

Change-Id: Icf26f1b84f5a653f541819a42980377f51c7f299
Signed-off-by: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
Signed-off-by: Stanislaw Kardach <Stanislaw.Kardach@cavium.com>
---
 build/f_isoroot/f_repobuild/Makefile | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/build/f_isoroot/f_repobuild/Makefile b/build/f_isoroot/f_repobuild/Makefile
index e95dbf2..3d1b566 100644
--- a/build/f_isoroot/f_repobuild/Makefile
+++ b/build/f_isoroot/f_repobuild/Makefile
@@ -12,6 +12,7 @@ SHELL := /bin/bash
 TOP := $(shell pwd)
 TMP_ROOT_DIR := $(shell echo "$(MIRROR_UBUNTU_ROOT)" | cut -d "/" -f2)
 # Enable use of exact repo commit if defined, else use Fuel main branch
+FUEL_MIRROR_REPO?=https://github.com/openstack/fuel-mirror
 FUEL_MIRROR_COMMIT?=$(FUEL_MAIN_TAG)
 
 include ../../config.mk
@@ -26,7 +27,7 @@ nailgun:
 	sudo apt-get install -y git libxml2-dev libxslt-dev python-dev  python-pip libz-dev libyaml-dev createrepo python-yaml
 	rm -Rf nailgun
 	sudo mkdir -p /var/www/nailgun
-	git clone https://github.com/openstack/fuel-mirror
+	git clone $(FUEL_MIRROR_REPO)
 	cd fuel-mirror && git checkout -q $(FUEL_MIRROR_COMMIT)
 	sudo pip install -U -r ./fuel-mirror/requirements.txt
 	sudo pip install ./fuel-mirror
-- 
cgit v0.12

